src/dproto_tcp.erl

```diff
@@ -418,6 +444,14 @@ decode(<<?GET,
          Time:?TIME_SIZE/?SIZE_TYPE, Count:?COUNT_SIZE/?SIZE_TYPE>>) ->
     {get, Bucket, Metric, Time, Count};
 
+decode(<<?GET,
+         _BucketSize:?BUCKET_SS/?SIZE_TYPE, Bucket:_BucketSize/binary,
+         _MetricSize:?METRIC_SS/?SIZE_TYPE, Metric:_MetricSize/binary,
+         Time:?TIME_SIZE/?SIZE_TYPE, Count:?COUNT_SIZE/?SIZE_TYPE,
+         RR:?GET_OPT_SIZE/?SIZE_TYPE, R:?GET_OPT_SIZE/?SIZE_TYPE>>) ->
+    Opts = [decode_opt({rr, RR}), decode_opt({r, R})],
```

Comment:
any reason not to use two functions here? They will have a smaller size each and be more readable that way?

`decode_rr(RR) -> {rr, ...}`